css/style.css

```diff
@@ -10,6 +10,7 @@ ul {
 li {
     width: 25%;
     color: aqua;
+    background-color: bisque;
 }
 #navbar li{
     list-style: none;
```

Comment:
Please changes the `list-style: none;` to some bullet points and also apply some background color. Follow this link [https://www.w3schools.com/css/css_background.asp](url); for further how to use the background color.

Reply:
> Please changes the `list-style: none;` to some bullet points and also apply some background color. Follow this link [https://www.w3schools.com/css/css_background.asp](url); for further how to use the background color.

- [ ] Same thing applies here.

Reply:
- [Optionnal] Kindly uses a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖.